torch/distributed/distributed_c10d.py

```diff
@@ -166,6 +198,29 @@ def get_default_group():
     return _default_pg
 
 
+def get_backend(group=group.WORLD):
```

Comment:
@pietern suggested that I return an enum. TBH I am fine with either...

Reply:
@apaszke OK with using string instread. I grepped the repo and there is no existing use of enums so it's probably better to steer clear of them and not start introducing them here. We can also change the named constants to be a string and then they are both comparable with whatever comes back from `get_backend` as well as printable.

Reply:
That sounds good to me. We've been in similar situations before, and the decision has always been to go with strings, because they are a lot easier to use in Python.